composer.json

```diff
@@ -9,19 +9,19 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "gedmo/doctrine-extensions": "^2.3",
-        "knplabs/knp-menu-bundle": "^3",
+        "php": "^7.2 || ^8.0",
+        "gedmo/doctrine-extensions": "^2.3 || ^3.0",
+        "knplabs/knp-menu-bundle": "^3.0",
         "symfony/console": "^4.4",
```

Comment:
Ik mis `symfony/http-kernel`. Behoorlijk relevant, aangezien dit een bundle is en daar `Symfony\Component\HttpKernel\Bundle\Bundle` in zit 😉 

Ook:
- `symfony/dependency-injection`
- `symfony/routing`
- `symfony/security`

Ook mis ik een dependency op `zicht/status-bundle` waar in `ValidateNestedTreeProvider` naar wordt gerefereerd

Reply:
Mja, die hebben er nooit in gestaan en binnen projecten waar de menu-bundle gebruikt wordt zijn deze kern-componenten van Symfony uiteraard wel aanwezig.

Ik heb `zicht/status-bundle` als suggestie toegevoegd, dat is optioneel en breekt verder geen code als die er niet is.

Reply:
> Ik heb `zicht/status-bundle` als suggestie toegevoegd, dat is optioneel en breekt verder geen code als die er niet is.

Dit breekt wel, want de class wordt genoemd in de service definities. Dus bij de eerste de beste cache clear krijg je een error:

https://github.com/zicht/menu-bundle/blob/1b35439126162392d747117ade6613046e2caf06/src/Resources/config/services.xml#L51-L54

> Mja, die hebben er nooit in gestaan en binnen projecten waar de menu-bundle gebruikt wordt zijn deze kern-componenten van Symfony uiteraard wel aanwezig.

Ik vind dit meer geluk dan wijsheid

Reply:
Mja er wordt op deze manier wel met cross-referenties gestrooid die we maar allemaal moeten administreren. Daarbij is de status-bundle ook een private repository... Je zou dit eigenlijk willen omdraaien en deze StatusProvider misschien wel in de StatusBundle willen hebben...

Ik ga in mijn test-setup even kijken wat er gebeurt als ik de status-bundle verwijder.

Reply:
👍🏻 Overigens is het de vraag of het erg is dat we in deze publieke zicht/menu-bundle de private zicht/status-bundle zouden requiren, want ik heb zeker niet de illusie dat iemand buiten Zicht deze bundles gebruikt.

Reply:
```
Symfony\Component\DependencyInjection\Compiler\RemoveUnusedDefinitionsPass: Removed service "zicht_bundle_menu.status_provider.validate_nested_tree_provider"; reason: unused.
```
Als ik de status-bundle weghaal verdwijnt deze service bij het compileren van de container omdat deze nooit gebruikt gaat worden. Wat dat aangaat dus "veilig".